data/json/items/gun/3006.json

```diff
@@ -90,6 +112,30 @@
     "magazines": [ [ "3006", [ "m1918mag", "m1918bigmag" ] ] ]
   },
   {
+    "id": "pipe_combination_gun",
+    "type": "GUN",
+    "name": "pipe combination gun",
+    "//": "Attached mod will now have weight.",
```

Comment:
This comment is useless here, and if we need to attach weight of this gun to some other weight it will be better to do it using ```copy-from``` trick.